ofrak_core/ofrak/core/apk.py

```diff
@@ -22,11 +22,11 @@
 from ofrak_type.range import Range
 
 
-APKTOOL = ComponentExternalTool("apktool", "https://ibotpeaches.github.io/Apktool/", "--help")
+APKTOOL = ComponentExternalTool("apktool", "https://ibotpeaches.github.io/Apktool/")
```

Comment:
#388 suggests using "" as the argument.

Is there any reason you decided against this? It seems preferable, as it doesn't change the signature of ComponentExternalTool.

Reply:
Could probably also use `"-version"`

Reply:
Even though the empty string works here (I did try it), not all CLI applications will ignore or properly handle an empty string argument. It feels more correct to me that if we don't want to pass an additional argument, we use `None` rather than the empty string. 

In this case, I wasn't worried about changing the signature since the same parameters are still accepted, albeit one is now a keyword argument. As a compromise, I could make that parameter typed as `Optional`, but not be a keyword argument with a default value. That way the signature changes even less, but still enables what I consider to be more robust, correct behavior. 

Thoughts @whyitfor?